src/js/modal/login-wallet/modal.js

```diff
@@ -85,7 +85,12 @@ export default Modal.extend({
 
 		this.$("#wallet-password").on("keyup change click", () => {
 			let password = this.$("#wallet-password").val();
-			let strength = zxcvbn(password);
+			if (password.length >= 128) {
+				let score = 5;
+			} else {
+				let strength = zxcvbn(password);
+				let score = strength.score;
+			}
```

Comment:
This can't be right. `let` variables do not exist outside of their scope like `var`s do. I think you mean:
```JavaScript
const score = password.length >= 128 ? 5 : zxcvbn(password).score;
```

Reply:
Ahhh you beat me